packages/hooks/src/useCurrentAccountDetails.ts

```diff
@@ -0,0 +1,10 @@
+import { useStore } from '@deriv/stores';
```

Comment:
with this hook, do we need this in the `withdraw-store` any more?
<img width="624" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/c81e2800-99db-4d53-be5c-56693ac5491d">

Reply:
Fixed 👍